src/lib/BarStore.js

```diff
@@ -12,31 +12,119 @@ const CATEGORIES = ['cozy', 'partners', 'ptnb']
 export default class BarStore {
   constructor () {
     this.claudyActions = null
+    this.installedApps = [] // to cache already fetched apps icons
+    this.helpLink = ''
   }
 
-  getClaudyActions () {
-    if (this.claudyActions) return Promise.resolve(this.claudyActions)
-    return stack.get.context()
-      .then(context => {
-        const contextActions = (context.data && context.data.attributes && context.data.attributes['claudy_actions']) || null
-        if (!contextActions) return null
-        // get an arrays of action
-        const claudyActions = contextActions.map(slug => {
-          if (CLAUDY_ACTIONS.hasOwnProperty(slug)) {
-            // adding also the action slug
-            return Object.assign({}, CLAUDY_ACTIONS[slug], { slug })
+  async fetchApps () {
+    let apps
+    try {
+      apps = await Promise.all((await stack.get.apps())
```

Comment:
Hum, it was already here, without that it seems to break so I will investigate this later, not in this PR.